typings/literally-canvas-defs.d.ts

```diff
@@ -0,0 +1,14 @@
+// Code - node_modules/literallycanvas/lib/js/literallycanvas-core.js
+
+
```

Comment:
This is not needed since these typings are not for this lib. Also maybe change the name of this file to custom-hash-node-map-defs.d.ts or something similar

Reply:
Done.